web/styles/banners.css

```diff
@@ -14,22 +14,6 @@
     border-radius: 6px;
 }
 
-.banner-link {
-    color: var(--color-text-link);
-    text-decoration: none;
-    text-decoration-color: var(--color-text-link-decoration);
-
-    &:hover {
-        color: var(--color-text-link-hover);
-        text-decoration-color: var(--color-text-link-decoration-hover);
-    }
-
-    &:active {
-        color: var(--color-text-link-active);
-        text-decoration-color: currentcolor;
-    }
-}
```

Comment:
I think we probably want to keep these styles; we will convert other banners that use links. So dropping this hunk. @sayamsamal FYI.